msvc-specific.cmake

```diff
@@ -29,6 +31,7 @@
 #  - or newer than MSVC 14.0
 #
 MACRO(GET_MSVC_VERSION)
+  message("This macro is deprecated. Use GET_MSVC_TOOLS_VERSION instead." DEPRECATION)
```

Comment:
Just a minor comment. I have always seen `message` used like `message(mode message)` and the documentation states the API of message is like this.

If you called it this way, I am sure it works. But it has puzzled me enough for me to check the doc. Do you have a specific preference for this one ?

Reply:
No preference. Previous `message` in the files were like this, so I kept it coherent.
I switched the order in 5eef6a3